django-backend/fecfiler/transactions/views.py

```diff
@@ -108,12 +106,12 @@ def create(self, request, *args, **kwargs):
             saved_transaction = self.save_transaction(request.data, request)
             print(f"transaction ID: {saved_transaction.id}")
         # transaction_view = self.get_queryset().get(id=saved_transaction.id)
-        return Response(TransactionSerializer().to_representation(saved_transaction))
+        return Response(saved_transaction.id)
 
     def update(self, request, *args, **kwargs):
```

Comment:
we'll want to change `partial_update`, `save_transactions`, and `save_reatt_redes_transactions` to just return ids.

They all call `save_transaction()` which has this return statement: 
```
return self.queryset.get(id=transaction_instance.id)
```
It runs the query again so it can get the correct just-in-time values.  Since we're not going to return the transaction in the payload, what do you think of changing `save_transaction` to return just the id?